guides/v2.2/rest/bulk-endpoints.md

```diff
@@ -10,8 +10,8 @@ functional_areas:
 
 Bulk API endpoints differ from other REST endpoints in that they combine multiple calls of the same type into an array and execute them as a single request. The endpoint handler splits the array into individual entities and writes them as separate messages to the message queue.
 
-{:.bs-callout .bs-callout-tip}
-Use the `bin/magento queue:consumers:start async.operations.all` command to enable bulk endpoint processing.
+{:.bs-callout-info}
+Use the [`bin/magento queue:consumers:start async.operations.all`](https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-queue.html) command to start the consumer that handles asynchronous and bulk API messages. Also, before using the Bulk API to process messages, you must install and configure RabbitMQ, which is the default message broker. See [RabbitMQ]({{ page.baseurl }}/install-gde/prereq/install-rabbitmq.html).
```

Comment:
Please also lets use always same writing standard: Bulk API and Asynchronous (First capital). Pls check everywhere

Reply:
> Please also lets use always same writing standard: Bulk API and Asynchronous (First capital). Pls check everywhere

You are correct. We aren't consistent in our capitalization.  However, this issue is outside scope of this PR.  Suggest opening a new PR to review and update occurrences of references to "bulk API" and "asynchronous" as needed.  When we reference "Bulk API" as a feature name makes sense to capitalize, but sometimes we have references more generic references like bulk API or asynchronous messages, which do not need to be capitalized.

Reply:
Agree)) but maybe we have to start from somewhere 😂